AnaMunoz-Fatima-Functions-0826.js

```diff
@@ -0,0 +1,132 @@
+//============================================
+// PART 2 
+//============================================
+
+
+// # 1 Calculate the Cube
+//============================================
+const calculateCube = (x) => {
+  return x**3;
+}
+console.log(calculateCube(5));
+
+
+// #2 Is a Vowel?
+//============================================
+const isAVowel = (letter) => {
+  if (letter === 'a') {
+  return true
+} 
+else {
+return false
+}
+}
+console.log(isAVowel("a"));
+// console.log(isAVowel("b"));//Test
```

Comment:
this works for the lowercase letter 'a' and nothing else. beef up your conditional so that it works for any vowel in any case.